strawberryfields/gbs/plot.py

```diff
@@ -252,11 +259,94 @@ def plot_subgraph(subgraph: nx.Graph, size: int = 500) -> None:  # pragma: no co
         xaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
         yaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
         margin=dict(b=0, l=0, r=0, t=25),
-        height=size,
-        width=size,
+        height=plot_size,
+        width=plot_size,
         plot_bgcolor="#ffffff",
     )
 
     f = go.Figure(data=[g_edges, g_nodes], layout=layout)
 
     return f
+
+
+def plot_points(R: np.ndarray, sample: Optional[list] = None,
+                plot_size: int = 500, point_size: int = 30) -> None:  # pragma: no cover
```

Comment:
```suggestion
                plot_size: int = 500, point_size: int = 30) -> go.Figure:  # pragma: no cover
```
I noticed this is also the case for the other 2 plotting functions

Reply:
Since we're not importing plotly directly in the module but through a `try`, this raises an error. Any suggestion on what to do?